src/docs/Foundations/Typography/HeadingStyles/code.mdx

```diff
@@ -3,81 +3,124 @@ title: 'Code'
 order: 2
 ---
 
-## Base text classes
+## Import Headings
 
-Base typography classes **does not apply any external margins nor font-colors**, because they are contextuals. This mean that **color and margins will be either inherited or applied through new classes** set on your dom elements, depending on their position and usage.
+Headings sizes change at the `M` and the `L` viewport, witch correspond to tablet and landscape tablets/laptops sizes.
 
-You can import all base typography scss files by importing the `_a.all-base-typography` sub-bundle from the `atoms` directory
+@import 'typography/_t.headings';
+```
+
+## Usage
+
+<Highlight theme="warning" title="Keep in mind">
+
+You need to apply **at least two classes**, the block name and a size.
+
+</Highlight>
 
-```scss
-@import 'settings-tools/_all-settings';
 
-@include import-font-families();
 
-@import 'typography/_t.all-base-typography';
+```html
+<p class="mt-heading mt-heading--m">Lorem ipsum...</p>
 ```
 
-## Heading classes
+## Variations
 
-> Default lines-heights for the headings familly are set to `M`, witch correspond to a ratio close to 1.35.
-> Each heading atom comes with **a modifier to set the font-weight to 'semi bold'** and **a modifier to use an alternative `S` line-height**.
-> Headings atoms sizes change at the 'M' viewport, witch correspond to tablet sizes.
+### Sizes
 
-#### Import Headings atoms
+There is 3 Headings sizes available. Each heading size rescale it's font-size at the **M** and **L** viewports.
+Use the size modifiers in combination to the `mt-heading` class :
 
-```scss
-@import 'settings-tools/_all-settings';
+- **Small** : `mt-heading mt-heading--s`
+- **Medium** : `mt-heading mt-heading--m`
+- **Large** : `mt-heading mt-heading--l`
 
-@include import-font-families();
+<Preview path="typographyHeading-sizes" />
```

Comment:
Not really important, but it could be a good thing to putthe content of the preview in the same order as the introduction and the table: HeadingS before, etc...